packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -27,6 +27,10 @@ import 'debugger_model.dart';
 import 'scripts.dart';
 import 'variables.dart';
 
+// This is currently a dev-time flag as the overall call depth may not be that
+// interesting to users.
+const bool showCallStackCount = false;
```

Comment:
nit: rename to `debugShowCallStackCount` to make it clear that this flag is for debugging purposes

Reply:
changed